pkg/rego_vm/exceptions.yaml

```diff
@@ -1,15 +1,2 @@
 "baseandvirtualdocs/enumerate virtual errors": test case has both expected error and result
 "baseandvirtualdocs/base/virtual: conflicts": compile error in PrepareForEval
-"functions/default": unsupported in topdown
```

Comment:
Was this (and the other exception lines following) just legacy cruft from the early days? I could've sworn we had ref-heads support for a while. 🤔

Reply:
Yeah we've had this for long, @johanfylling did all the heavy lifting for opa's wasm. We just piggybacked on that.